content/tutorials/kit-assembly.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Connecting your kit
+---
+
+## The bits you should have
+- Raspberry Pi (the board in a plastic case with many USB ports)
+- Power Board (the board in a plastic case with many green sockets and two buttons)
+- Motor Board (the other board in a plastic case)
+- Arduino (The blue, exposed board with a metal USB-B connector)
+- Arduino Servo Shield (another blue board, hopefully on top of the first)
+- 1 x 3.81mm Camcon (for the Arduino Servo Shield)
+- 2 x 5mm Camcon (for the Motors)
+- 1 x 5mm Camcon with wire loop (attached to the power board)
+- 2 x 7.5mm Camcon (for the Motor Board)
+- USB Hub
+- USB Webcam
+- 2 x Black USB-A to micro-usb cables
```

Comment:
@RealOrangeOne there's a colour to remove here too (sorry, should have been clearer earlier).